packages/ag-charts-community/src/chart/agChartV2.ts

```diff
@@ -79,7 +81,7 @@ export abstract class AgChart {
     /**
      * Create a new `AgChartInstance` based upon the given configuration options.
      */
-    public static create(options: AgChartExtendedOptions): AgChartInstance {
+    public static create(options: AgChartExtendedOptions): AgChartProxy {
```

Comment:
`AgChart.create` and `AgChartInstance` are public, and I don't really want to advertise that it's a proxy as that is an implementation detail